docs/source/operations/orchestrator/deployment.rst

```diff
@@ -0,0 +1,223 @@
+
+   .. code-block:: bash
+
+      helm repo add substra https://substra.github.io/charts/
+
+#. Create a Helm values file named ``orchestrator-values.yaml`` with the following content:
+
+   .. code-block:: yaml
+
+      ingress:
+        enabled: true
+        hostname: HOSTNAME
+
+   | Replace ``HOSTNAME`` with the hostname of your Orchestrator.
+   | This will setup the ingress in order to make your Orchestrator accessible at the defined hostname.
+
+#. Setup your :term:`Substra channels<Channel>`.
+   In the ``orchestrator-values.yaml`` file, add the following content:
+        
+   .. code-block:: yaml
+
+      channels:
+        - name: CHANNEL-NAME
+          organizations: [ORG-NAME, ...]
+
+   | Replace ``CHANNEL-NAME`` with the name you want to give to your channel.
+   | Replace ``ORG-NAME, ...`` with the names of the :term:`Organizations<Organization>` that you want to participate in this channel.
+   | If you want more than one channels you can add more items in the list.
+
+Here you have created the most basic configuration required to have a running orchestrator that can support a Substra network.
+If you want you can jump directly to the section deploy or you can follow along the next sections to enhance the security of your orchestrator.
+
+Setup TLS for the Orchestrator
+------------------------------
+
+If you want to you can enable TLS for your orchestrator.
+For this you will need to generate a few certificates, here we will generate them manually but you can also use automated tools for this task.
+If you want to use automated tools we provide a certificate resource for `cert-manager <https://cert-manager.io/>`_, check out the ``orchestrator.tls.createCertificates`` values.
+
+To setup TLS, follow these steps:
+
+#. Enable TLS, in the ``orchestrator-values.yaml`` file add the following content:
+
+   .. code-block:: yaml
+
+      orchestrator:
+        tls:
+          enabled: true
+
+#. Generate a Certificate Authority certificate:
+
+   #. Create an openssl config file named ``example-openssl.cnf`` with the following content:
+
+      .. code-block:: ini
+
+         [ req ]
+         default_bits		= 2048
+         default_md		= sha256
+         distinguished_name	= req_distinguished_name
+
+         [ req_distinguished_name ]
+
+         [ v3_ca ]
+         basicConstraints = critical,CA:TRUE
+         subjectKeyIdentifier = hash
+         authorityKeyIdentifier = keyid:always,issuer:always
+         keyUsage = cRLSign, keyCertSign
+
+   #. Generate a private key for signing certificates:
+
+      .. code-block:: bash
+
+         openssl genrsa -out orchestrator-ca.key 2048
+
+   #. Generate your Certificate Authority certificate:
+        
+      .. code-block:: bash
+
+        openssl req -new -x509 -days 365 -sha256 -key orchestrator-ca.key -extensions v3_ca -config example-openssl.cnf -subj "/C=FR/ST=Loire-Atlantique/L=Nantes/O=ORC-ORG-NAME/CN=Orchestrator Root CA" -out orchestrator-ca.crt
```

Comment:
Can we leave the country, state and location out?

Reply:
Removed them.